kpatch-build/create-diff-object.c

```diff
@@ -872,6 +886,9 @@ static void rela_insn(struct section *sec, struct rela *rela, struct insn *insn)
 			return;
 	}
 }
+#else
+void rela_insn(struct section *sec, struct rela *rela, struct insn *insn) { }
+#endif
```

Comment:
I think this empty prototype isn't needed since it's only called from x86-specific code.